package.json

```diff
@@ -57,6 +57,7 @@
     "@types/chai-as-promised": "^7.1.4",
     "@types/mocha": "^8.2.2",
     "@types/node": "^12.0.0",
+    "@types/pino-pretty": "^5.0.0",
```

Comment:
This doesn't feel relevant to the PR.

Reply:
it was required for me to get running on mac, but can remove